packit/config/config.py

```diff
@@ -283,7 +283,7 @@ def get_default_map_from_file() -> Optional[dict]:
     return None
 
 
-@lru_cache()
+@lru_cache
```

Comment:
Now, here I'm not sure if it's the same :eyes:

Reply:
See the [example](https://docs.python.org/3.9/library/functools.html#functools.lru_cache)